.github/workflows/coverage-report.yml

```diff
@@ -0,0 +1,25 @@
+name: Get Code Coverage Report
```

Comment:
It would be good to just give a brief purpose of this workflow file at the top for future maintainers! 

An example can be seen [here](https://github.com/CATcher-org/CATcher/blob/master/.github/workflows/deployment-staging.yml) !